ros_buildfarm/templates/snippet/publisher_extended-email.xml.em

```diff
@@ -40,6 +40,7 @@
       <attachBuildLog>false</attachBuildLog>
       <compressBuildLog>false</compressBuildLog>
       <replyTo>$DEFAULT_REPLYTO</replyTo>
+      <from/>
```

Comment:
I don't think this is related to specific plugin updates as I don't see
any of the mail related plugins in recent diffs however since part of
this upgrade was a major change to xml serialization it's possible that
things have changed there. The empty "from" does give me a little bit of
worry. We'll have to watch this one when it is deployed.